libs/libarchfpga/src/physical_types.h

```diff
@@ -624,6 +642,7 @@ struct t_physical_tile_type {
     /* Returns the indices of pins that contain a clock for this physical logic block */
     std::vector<int> get_clock_pins_indices() const;
 
+    // Returns the sub tile location (z coordinate) given an input pin of the physical tile
```

Comment:
We no longer have the z coordinate notation. We can change the comment to be:
// Returns the sub tile location of the physical tile given an input pin

Reply:
Done